src/app_engine/apprtc.py

```diff
@@ -590,6 +590,20 @@ def get(self):
     params = get_room_parameters(self.request, None, None, None)
     self.response.write(json.dumps(params))
 
+def checkIfRedirect(self):
+  REDIRECT_DOMAINS = [
+    'apprtc.appspot.com', 'apprtc.webrtc.org', 'www.appr.tc'
```

Comment:
Out of curiosity I also tried www.apprtc.appspot.com and it gives me an "invalid certificate" error. Is that something we want to fix?